web/impact/impact/permissions.py

```diff
@@ -33,6 +33,15 @@ def has_permission(self, request, view):
             name=settings.V1_API_GROUP).exists()
 
 
+class V1ConfidentialAPIPermissions(BasePermission):
+
```

Comment:
what is the purpose of this permission? I'm not seeing what necessitates a new permission here. If our v1 permission class is not robust enough to support read-only type access (I think we should invest in adding that capability instead of splitting out a new group+permission). thoughts?

Reply:
John Harthorne wants the confidential information to be separately controlled from the other calls for future use of the other calls to folks who are not concerned about that data.  This was a late breaking requirement.

Reply:
ok i see - is there any accelerate-side permission that already exists that we could latch onto? If not i suppose creating its own group is a reasonable way of segmenting this. - i dont want this point to block getting this out since it seems to work as-is

Reply:
Don't think there's an equivalent accelerate-side permission.